### ISSUE TITLE
Godot crashes when reimported file is visible in editor preview

### ISSUE THREAD

**Issue by m0nkeybusin3ss** [viniciusgerevini/godot-aseprite-wizard#174]
Hello viniciusgerevini and aseprite wizard community!
I have encountered a bug when reimporting a file that is currently opened in the editor. I have attached a video to showcase the problem. I imagine this bug is somehow hardware dependent, because I cannot think that this wouldn`t have brought up before, if everyone had this. I am working on an ASUS Zenbook DUO (2024) UX8406. It has Intel Arc Graphics and no dedicated graphics card. Maybe this contributes to the bug.

I have checked the Issues and the docs for something similar to this issue but could not find anything. Sorry if I missed something. I do not know which log files to attach, so if you need any further assistance or information, please let me know!

https://github.com/user-attachments/assets/bf852447-40a7-4d29-86f1-f31c4d441870

Cheers

**Comment by viniciusgerevini** [comment]
Thanks @m0nkeybusin3ss . I was able to reproduce a similar issue. The behaviour is slightly different than yours, but I think it's the same root cause.

On my machine the issue only happens with the GPUParticles2D node, and when trying to use the `.png` file instead of the `.aseprite` one. The png file is replaced externally on import, and the Godot Editor is probably crashing due to the extra scan required.

I suggest you trying to use the `.aseprite` file instead of the `.png` file. Ideally, no png file should be generated (#163), but unfortunately due to limitations in the import system, I can't hide it at the moment.

I have opened a discussion in the godot proposals for that (https://github.com/godotengine/godot-proposals/discussions/11042), but it hasn't received much attention. Feel free to upvote it and even comment on it if you are interested. 

Let me know in case using the `aseprite` file doesn't work in your system.

**Comment by KeyboardDanni** [comment]
Related to https://github.com/viniciusgerevini/godot-aseprite-wizard/issues/157?

> I suggest you trying to use the `.aseprite` file instead of the `.png` file. Ideally, no png file should be generated ([#163](https://github.com/viniciusgerevini/godot-aseprite-wizard/issues/163)), but unfortunately due to limitations in the import system, I can't hide it at the moment.

Up to this point I've been using the .png file directly, but I've wondered if I should use the .aseprite file instead. My main concern is it's unclear to me if the raw .aseprite file itself will be included in the game's pack file. For my projects I've been specifically excluding *.aseprite files when exporting the game. If the export just includes the .ctex and Godot-related data (such as tileset/sprite frame data), I'd be fine with that.

**Comment by viniciusgerevini** [comment]
> If the export just includes the .ctex and Godot-related data (such as tileset/sprite frame data), I'd be fine with that.

Yes, I believe this is how it works. When importers are enabled, references to the source file will actually point to the generated file (usually the ones inside `.godot`/`.import`, but in this plugin's case, these files are the .res / .png generated alongside the main file).

I haven't checked if when exporting the game the original aseprite files are still kept inside the pack. I assume they are removed to save space. But even if you explicitly remove them from the export, the game resources should already be referencing the generated files and the game should work as expected.

**Comment by viniciusgerevini** [comment]
Version 9.0.0 does not expose the extra PNG file anymore, so this issue shouldn't be reproducible anymore. Let me know if you see the same error using the .aseprite file. Cheers
